### DISCUSSION TITLE
Filaments

### DISCUSSION THREAD

**Discussion by krisrok** [Sakati84/3DPrintColorizer#80] [General]
First of all hi and thanks for all your work on this project! I'm still in preparation and as I need to order filament anyway I find it hard to choose a supplier.

I wonder which ones you've already tried and what your results were (maybe even with pictures)?

In the existing [wiki page](https://github.com/Sakati84/3DPrintColorizer/wiki/2-Materials#recommended-filament) it's kind of vague right now. Ok, the Basics PLA seems to work best but... 
- What specific brands did not work? For what reasons?
- Have you tried PETG, ABS or even more exotic materials?
- What about "natural" PLA? Is this transparent enough?
- And kind of related: What about post-processing the prints? Clear coating, smoothing, sanding...?

Of course this targets @Sakati84 first and foremost but I guess we need a place to document our findings. Maybe we can collect some data here and then move the important bits into the wiki?

**Comment by Sakati84** [comment]
Janbex didn´t work for me. I couldn´t get it to print ok without coloring either..
I think natural PLA would work great. It doesn´t need to be very transparent. It should be a little "milky" for best results.
